runtime/doc/nvim_python.txt

```diff
@@ -24,10 +24,15 @@ simple tutorial:
 
 - Make sure Python 2.6 or 2.7 is available in your `$PATH`
 - Install the `neovim` Python package:
-  >
-      $ pip install neovim
+>
+    $ pip install neovim
```

Comment:
Not regarding the indentation: We should probably update this to `pip install --user neovim` as to match the wiki, although that might be unfair because I was the one who changed it...

Still, `$` is generally representative of a prompt for an unprivileged user, while `#` is for root, so the command in question will fail if `--user` isn't passed to it.